allensdk/brain_observatory/vbn_2022/metadata_writer/schemas.py

```diff
@@ -81,3 +82,49 @@ def validate_paths(self, data, **kwargs):
                 f"{msg}"
                 "Run with clobber=True if you want to overwrite")
         return data
+
+
+class PipelineMetadataSchema(DefaultSchema):
+
+    name = argschema.fields.Str(
+            required=True,
+            allow_none=False,
+            description=(
+                "Name of the pipeline component (e.g. 'AllenSDK')"))
+
+    version = argschema.fields.Str(
+            required=True,
+            allow_none=False,
+            description=(
+                "Semantic version of the pipeline component"))
+
+    comment = argschema.fields.Str(
+            required=False,
+            default="",
+            description=(
+                "Optional comment about this piece of software"))
+
+
+class VBN2022MetadataWriterOutputSchema(argschema.ArgSchema):
```

Comment:
Are you really sure this is specific to VBN 2022?

Reply:
Good catch. It's not.

Since we haven't yet committed to my harebrained "put all of the data release generation tools in one place" scheme, I'm not sure where to put it. I've renamed the class to be more descriptive, but I'm leaving it here. When/if we decide to consolidate our data release generation tools, we can move it over.

Does that seem like a compromise solution?